frontend/src/pages/devices/List.tsx

```diff
@@ -216,28 +215,16 @@ export const List = () => {
                         const minutesBetween = (wc - sa) / 1000 / 60;
                         if (expectedResponseInMinutes < minutesBetween) {
                             const distance = formatDistance(sa, wc);
-                            warnings.push(<Badge variant="danger">Slow to Respond (took { distance } to add code { lc.code })</Badge>);
+                            warnings.push(<p>Slow to Respond (took { distance } to add code { lc.code })</p>);
```

Comment:
Do these need to be `<p>`s? What does it look like without them?

Reply:
Note that you might need to have a container element, there's something you can do with JSX that puts a container around your text that isn't really a container (`<>hi</>`).

Reply:
But it might be better to keep it in a `Badge` if most of the text in the other columns is a `Badge` - that way it all looks the same.